starwars/src/components/Card.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './StarWars.css'
+
+function Card(props) {
+    const data = props.data
```

Comment:
Pro tip - You can use destructuring here like:

> const { data } = props

It is doing the exact same thing as when we use `import React { Component } from 'react';` to be able to write `class X extends Component`.